fuse/plugins/detector_physics/s1_photon_propagation.py

```diff
@@ -374,14 +410,23 @@ def nest_scintillation_timing(
 
             loop_count = 0
             while n_times_to_sample > 0:
-
-                scint_time = self.nestpy_calc.GetPhotonTimes(
-                    nestpy.INTERACTION_TYPE(recoil_type[i]),
-                    n_times_to_sample,
-                    n_excitons[i],
-                    local_field[i],
-                    e_dep[i],
-                )
+                # If NR types, we check if we want to use the effective scintillation delay model
+                if self.override_s1_nr_scint_time and (recoil_type[i] <= 6):
```

Comment:
@ramirezdiego please take a look if this makes sense to you. I was following 
```
    nr=0, wimp=1, b8=2, dd=3, ambe=4, cf=5, ion=6, gammaray=7,
    beta=8, ch3t=9, c14=10, kr83m=11, nonetype=12
```
from my old knowledge of NEST types